index.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: link
 active_page: link
-title: Ableton @ GitHub
+title: Ableton on GitHub
```

Comment:
I'm not sure that this change will impact SEO much. Do you have any insight on this?

Reply:
I was not thinking about the SEO but it just looked better to me and might be easier for screen readers.